portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
```

Comment:
There is a shorthand when the left and right padding are the same
https://www.w3schools.com/css/css_padding.asp
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

Reply:
done